examples/hello_world_serde.rs

```diff
@@ -0,0 +1,65 @@
+use bevy::prelude::*;
+use bevy_input_actionmap::*;
+fn main() {
+    App::build()
+    .add_plugins(DefaultPlugins)
+    .add_plugin(ActionPlugin::<Action>::default())
+    .add_startup_system(setup.system())
+    .add_system(run_commands.system())
+    .run();
+}
+
+#[derive(Hash, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serialize", derive(Serialize, Deserialize))]
+enum Action {
+    Select,
+    SuperSelect,
+    AwesomeSuperSelect,
+}
+
+#[cfg(feature = "serialize")]
+fn setup(mut input: ResMut<InputMap<Action>>) {
+    if let Err(_) = input.load_from_path("keybinds.config") {
```

Comment:
```suggestion
    if let Err(_) = input.load_from_path("keybindingss.config") {
```

I like "keybindings" a bit better than "keybinds". :)